### PR TITLE
Update rules.bolt

### PR DESCRIPTION
**Pull request by aleksrozman**
Based on the latest sample data to include notifications, maps, markers, and lives

### CODE REVIEW

File: firebase/rules.bolt [hunk 1]
```diff
@@ -85,6 +92,10 @@ getUserId() {
   ('user-' + auth.uid)
 }
 
+getPrivateLifeId() {
+  ('privateLife-' + auth.uid)
```
Comment (Verdagon) [inline firebase/rules.bolt]: alas, we cant make this assumption =(

a player might go through multiple lives throughout the game, and each one will have a different ID, and a different code.

File: firebase/rules.bolt [hunk 2]
```diff
@@ -275,20 +286,35 @@ type NotificationSettings {
...
   notes: String;
   notificationSettings: NotificationSettings;
+  notifications: Map<NotificationId, Notification>;
   userId: UserId;
   volunteer: Volunteer;
   wantToBeSecretZombie: Boolean;
-  chatRooms: Map<ChatRoomId, Boolean>;
-  missions: Map<MissionId, Boolean>;
-  lives: Map<LifeId, PrivateLife>
+  chatRoomMemberships: Map<ChatRoomId, ChatRoomLookup>;
```
Comment (Verdagon) [inline firebase/rules.bolt]: can we call this ChatRoomLookup a ChatRoomMembership? thanks!

File: firebase/rules.bolt [hunk 3]
```diff
@@ -322,16 +348,17 @@ type Reward {
...
+  PublicPlayerId: PublicPlayerId; // #1
+  isActive: Boolean;
   allegiance: Allegiance;
   claims: Map<RewardId, Reward>;
   name: String;
   number: Number;
   points: Number;
   profileImageUrl: HtmlUrl;
   userId: UserId;
-  lives: Map<LifeId, PublicLife>
```
Comment (Verdagon) [inline firebase/rules.bolt:334]: accidental? there should still be a lives in publicplayer

File: firebase/rules.bolt [hunk 4]
```diff
@@ -426,11 +521,13 @@ type PublicLife extends LifeBase {
 /groups/{GroupId} is Group;
 /groups/GroupId/players/{PlayerId} is PrivatePlayer;
 /guns/{GunId} is Gun;
+/queuedNotification/{QueuedNotificationId} is QueuedNotification;
+/maps/{MapId} is MapType;
 /missions/{MissionId} is Mission;
 /playersPrivate/{PlayerId} is PrivatePlayer;
```
Comment (Verdagon) [inline firebase/rules.bolt]: in the firebase, this was renamed to privatePlayers and publicPlayers (to be consistently the same name as their type)

**Review by Verdagon — CHANGES_REQUESTED** [review]

**Comment by Verdagon** [comment]
also, can you see if backend_test runs well with these in place? thanks!

**Comment by aleksrozman** [comment]
Made the changes. Backend test run fine, but they are the server and thus it will work. Front end looks fine too (seems odd, maybe my config isn't running?)

**Review by Verdagon — APPROVED** [review]

**Pull request merged**